kmk/extensions/peg_oledDisplay.py

```diff
@@ -0,0 +1,133 @@
+import busio
+import gc
+
+import adafruit_displayio_ssd1306
+import displayio
+import terminalio
+from adafruit_display_text import label
+
+from kmk.extensions import Extension
+
+
+class oled(Extension):
+    def __init__(
+        self,
+        board,
+        views,
+        toDisplay: str = 'TXT',
+        oWidth: int = 128,
+        oHeight: int = 32,
+        flip: bool = False,

```

Comment:
these inline type hints have to go in comment form rather than inline, sadly. the PEP that implements inline type hints isn't universally implemented in CircuitPython (last we checked in https://github.com/KMKfw/kmk_firmware/pull/226#issuecomment-903316710)